contrib/inventory/gce_googleapiclient.py

```diff
@@ -82,30 +87,37 @@ def get_all_billing_projects(billing_account_name, api_version='v1'):
     return project_ids
 
 
-def get_all_zones_in_project(project, api_version='v1'):
+def get_all_zones_in_project(project, queue, api_version='v1'):
     zones = []
 
     credentials = GoogleCredentials.get_application_default()
     service = discovery.build('compute', api_version, credentials=credentials)
-
+    print('get_all_zones_in_project, project: {}, queueid: {} '.format(project, id(queue)))
     request = service.zones().list(project=project)
     while request is not None:
-        response = request.execute()
+        try:
+            response = request.execute()
+        except HttpError as exc:
+            log.info('Problem with retrieving zones: %s', str(exc))
```

Comment:
Add project name to the log message. Also, I think warning level would be more appropriate here.